package.json

```diff
@@ -65,7 +65,7 @@
     "@nx/esbuild": "16.3.1",
     "@nx/eslint-plugin": "16.3.1",
     "@nx/jest": "16.3.1",
-    "@nx/js": "16.3.1",
+    "@nx/js": "16.3.2",
```

Comment:
Why was this updated but not the other packages?

Reply:
curious, I'm pulling everything up to `16.4.0`. Tested across both IDEs and it seems there were no problems.